packages/server/api/src/app/workers/engine-controller.ts

```diff
@@ -81,8 +81,8 @@ export const flowEngineWorker: FastifyPluginAsyncTypebox = async (app) => {
     })
 
     app.post('/create-trigger-run', CreateTriggerRunParams, async (request) => {
```

Comment:
Please delete this endpoint and from there, try to move trigger-run-service into server-shared where it takes redis connection in the paramater, and use it in both places. 

Please remove trigger-run entity (and run TRUNCATE trigger-run) on the tables to free up the database.